ci/release/update-version.sh

```diff
@@ -11,7 +11,7 @@
 NEXT_FULL_TAG=$1
 
 # Get current version
-CURRENT_TAG=$(git tag | grep -xE 'v[0-9\.]+' | sort --version-sort | tail -n 1 | tr -d 'v')
+CURRENT_TAG=$(cat VERSION)
```

Comment:
can you revert the `cat VERSION` change here and make sure this line matches the change that was done in #42 after you merge those changes into your PR?

Reply:
done!

Reply:
Actually, to make the statement more robust to tag changes from other branches (e.g., adding a patch tag[21.06.01] to branch-21.06 after creating tag[21.08.00]), 

```bash
CURRENT_TAG=$(git tag | grep -xE 'v[0-9\.]+' | sort --version-sort | tail -n 1 | tr -d 'v')
```
to
```
CURRENT_TAG=$(git tag --merged HEAD | grep -xE 'v[0-9\.]+' | sort --version-sort | tail -n 1 | tr -d 'v')
```
what do you think?

Reply:
sure, that seems like a good idea. feel free to add it to this PR.